src/xdocs/property_types.xml

```diff
@@ -325,16 +325,55 @@
           <td>Definition</td>
         </tr>
 
+        <tr>
+          <td><code>alone</code></td>
+          <td>
+            The brace must be alone on the line. For example:
+            <pre>
+    try {
+        ...
+    <b>}</b>
+    finally {
+        ...
+    <b>}</b>
+            </pre>
+          </td>
+        </tr>
+
+        <tr>
+          <td><code>alone_or_singleline</code></td>
+          <td>
+                  The brace must be alone on the line, yet
+                  single-line format of block is allowed.
+                  For example:
+            <pre>
+    // Brace is alone on the line
+    try {
+        ...
+    <b>}</b>
+    finally {
+        ...
+    <b>}</b>
+
+    // Single-line format of block
+    public long getId() { return id; <b>}</b>
+            </pre>
+          </td>
+        </tr>
+
         <tr>
           <td><code>same</code></td>
           <td>
-            The brace should be on the same line as the next part of a multi-block statement
+            Works like <code>alone_or_singleline</code> but the brace should be on the same line
+            as the next part of a multi-block statement
             (one that directly contains multiple blocks: if/else-if/else or try/catch/finally).
```

Comment:
https://github.com/checkstyle/checkstyle/pull/5987#discussion_r199563745
> changed both lines.

I don't really see any updates to explain behavior for no multi-block.

Reply:
added special sentence to make it clear.